aspnet/fundamentals/localization.rst

```diff
@@ -1,8 +1,302 @@
+
+`Localizability <https://msdn.microsoft.com/en-us/library/aa292135(v=vs.71).aspx>`__ is an intermediate process for verifying that a globalized application is ready for localization
+
+The format for the culture name is "<languagecode2>-<country/regioncode2>", where <languagecode2> is the language code and <country/regioncode2> is the subculture code. Examples include ``es-CL`` for Spanish (Chile), ``en-US`` for English (United States), and ``en-AU`` for English (Australia). See `Language Culture Name <https://msdn.microsoft.com/en-us/library/ee825488(v=cs.20).aspx>`__.
+
+Internationalization is often abbreviated to "I18N". The abbreviation takes the first and last letters and the number of letters between them, so 18 stands for the number of letters between the first "I" and the last "N". The same applies to Globalization (G11N), and Localization (L10N).
+
+The terms used so far:
+
+- Globalization (G11N): The process of making an application support different languages and regions.
+- Localization (L10N): The process of customizing an application for a given language and region.
+- Internationalization (I18N): Describes both globalization and localization.
+- Culture: It is a language and, optionally, a region.
+- Neutral culture: A culture that has a specified language, but not a region. (for example "en", "es")
+- Specific culture: A culture that has a specified language and region. (for example "en-US", "en-GB", "es-CL")
+- Locale: A locale is the same as a culture.
+
+ASP.NET 5 allows you to specify two culture values,  ``SupportedCultures`` and ``SupportedUICultures``. The `CultureInfo <https://msdn.microsoft.com/en-us/library/system.globalization.cultureinfo(v=vs.110).aspx>`__ object for ``SupportedCultures`` determines the results of culture-dependent functions, such as the date time, number, and currency formatting. ``SupportedCultures`` also determines the sorting order of text, casing conventions, and string comparisons. See `CultureInfo.CurrentCulture <https://msdn.microsoft.com/en-us/library/system.globalization.cultureinfo.currentculture%28v=vs.110%29.aspx>`__ for more info on how the server gets the Culture. The ``SupportedUICultures`` determines which resources are to be loaded for the page by the ResourceManager. The ResourceManager simply looks up culture-specific resources that is determined by ``CurrentUICulture``. Every thread in .NET has ``CurrentCulture`` and ``CurrentUICulture`` objects. ASP.NET Core inspects these values when rendering culture-dependent functions. For example, if current thread's culture  is set to "en-US" (English, United States), DateTime.Now.ToLongDateString() shows "	Thursday, February 18, 2016", but if CurrentCulture is set to "es-ES" (Spanish, Spain) the result will be "jueves, 18 de febrero de 2016"".
```

Comment:
`ASP.NET 5` -> `ASP.NET Core`
`,  ` (double space after comma)
`such as the date time, number,` -> `such as the date, time, number,` (or `date-time`/`datetime`/`date/time`, etc.)
`For example, if current thread's culture` -> `For example, if the current thread's culture`
`culture is set to` (double space after culture)
`shows "	Thursday` -> `shows "Thursday`
`febrero de 2016""` -> `febrero de 2016"`